app/[slug]/(base)/settings/members/page.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+	ExistingMembers,
+	InviteDialog,
+	PendingInvitations
+} from '~/components/dashboard/Settings'
+import prisma from '~/prisma'
+
+export default async function Members({params}: {params: {slug: string}}) {
+	const team = await prisma.team.findUnique({
+		where: {slug: params.slug},
```

Comment:
Potential vulnerability: can anyone access this by knowing `slug`? Eg. I go to `maige.app/.../sarims-secret-team-slug`

Reply:
Might need a `where: { ..., userId: user.id }` or similar

Reply:
Solid catch, fixed